content/en/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md

```diff
@@ -0,0 +1,164 @@
+of the clusters within the mesh.
+
+## Multicluster Istio setup
+
+There are a couple of [multicluster deployment models](/docs/ops/deployment/deployment-models/#multiple-clusters)
+supported by Istio. You can follow the [multicluster installation](/docs/setup/install/multicluster/) section to setup
+your multicluster Istio. For the purposes of this guide, any of those approaches will work, with the following
+caveat:
+
+**Ensure that a cluster-local Istio Prometheus instance is installed in each cluster.**
+
+Individual Istio deployments of Prometheus in each cluster are required to form the basis of cross-cluster monitoring by
+way of federation to a production-ready instance of Prometheus that runs externally or in one of the clusters.
+
+For multicluster deployments that use the `remote` profile, you must add the following to the `istioctl manifest` command:
+
+{{< text plain >}}
+--set addonComponents.prometheus.enabled=true
+{{< /text >}}
+
+Validate that you have an instance of Prometheus running in each cluster:
+
+{{< text bash >}}
+$ kubectl -n istio-system get services prometheus
+NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+prometheus   ClusterIP   10.8.4.109   <none>        9090/TCP   20h
+{{< /text >}}
+
+## Configure Prometheus federation
+
+### External production Prometheus
+
+There are several reasons why you may want to have a Prometheus instance running outside of your Istio deployment.
+Perhaps you want long-term monitoring disjoint from the cluster being monitored. Perhaps you want to monitor multiple
+separate meshes in a single place. Or maybe you have other motivations. Whatever your reason is, you’ll need some special
+configurations to make it all work.
+
+{{< image width="80%"
+    link="./external-production-prometheus.svg"
+    alt="Architecture of external Production Prometheus for monitoring multicluster Istio."
+    caption="External Production Prometheus for monitoring multicluster Istio"
+    >}}
+
+{{< warning >}}
+This guide does not cover securing access to Prometheus. It demonstrates providing connectivity to cluster-local Prometheus
+instances in a few simple ways. For production use cases, it is recommended to secure access to each Prometheus endpoint
+with HTTPS, as well as taking appropriate precautions such as using an internal load-balancer instead of a publicly-accessible
+endpoint and/or properly configuring firewall rules.
+{{< /warning >}}
+
+Istio provides a way to expose cluster services externally via [Gateways](/docs/reference/config/networking/gateway/).
+You can configure an ingress gateway for the cluster-local Prometheus, providing external connectivity to the in-cluster
+Prometheus endpoint.
+
+For each cluster, follow the appropriate instructions from the [Remotely Accessing Telemetry Addons](/docs/tasks/observability/gateways/#option-1-secure-access-https) task.
+Also note that you **SHOULD** establish secure (HTTPS) access.
```

Comment:
the `Remotely Accessing Telemetry Addons` page changed while this doc was in development. we should update this to reflect the current recommendations from that page -- everything going through port `80` and using `nip.io` domains.

Reply:
@douglas-reid I read that updated doc, which `nip.io` is not recommended in production settings. I'm not sure how to add descriptions here. Since we already pointing out to the secure access https section for the user to follow? Might not need to update anything? But yeah, need your more specific suggestions for this part.